### PR TITLE
added prompt objects to metrics

### PR DESCRIPTION
**Pull request by tinomaxthayil**
* Added prompt objects to metrics

### CODE REVIEW

**Comment by tinomaxthayil** [comment]
@shahules786 faithfulness and critique prompts are left

**Comment by shahules786** [comment]
I have raised PR to your branch. @tinomaxthayil

**Review by shahules786 — CHANGES_REQUESTED** [review]

File: src/ragas/metrics/_answer_relevance.py [hunk 1]
```diff
@@ -19,49 +20,33 @@
...
-{{"question":"What was the groundbreaking feature of the smartphone invented in 2023?", "noncommittal":true}}
-
-
-
-Answer:
-{answer}
-Context:
-{context}
-Output:"""  # noqa: E501
+QUESTION_GEN = Prompt(
```
[the remark this answers is not in the document]
Comment (tinomaxthayil) [reply src/ragas/metrics/_answer_relevance.py:22]: You want to change this to 1/0?

[the remark this answers is not in the document]
Comment (shahules786) [reply src/ragas/metrics/_answer_relevance.py:22]: Yes

**Review by shahules786 — APPROVED** [review]

**Review by jjmachan — APPROVED** [review]

**Pull request merged**